sdk/communication/test-resources.json

```diff
@@ -13,12 +13,38 @@
       "defaultValue": "communication",
       "type": "string"
     },
+        "communicationServicesEndpointSuffix": {
+            "defaultValue": ".communication.azure.com",
+            "type": "string"
+        },
+        "subscriptionId": {
+            "defaultValue": "[subscription().subscriptionId]",
+            "type": "string"
+        },
     "testApplicationOid": {
       "type": "string",
       "metadata": {
         "description": "The client OID to grant access to test resources."
       }
-    }
+    },
+           "tenantId": {
```

Comment:
The indentation here seems off.

Reply:
Oh, I thought indentation does not matter. Regardless, I will fix it.